Firebase/Messaging/FIRMessagingRemoteNotificationsProxy.m

```diff
@@ -17,7 +17,9 @@
 #import "FIRMessagingRemoteNotificationsProxy.h"
 
 #import <objc/runtime.h>
+#if TARGET_OS_IOS ||TARGET_OS_TV
 #import <UIKit/UIKit.h>
```

Comment:
Nit: UIKit import should not be required when `GULAppDelegateSwizzler.h` imported.

Reply:
Nice. Thanks!